kingpin/actors/aws/cloudformation.py

```diff
@@ -959,6 +959,18 @@ def _diff_params_safely(self, remote, local):
             remote = [pair for pair in remote if pair['ParameterKey'] != p]
             local = [pair for pair in local if pair['ParameterKey'] != p]
 
+        # Remove any resolved parameter values that were inserted by SSM
+        # so that only supplied parameter values are compared.
+        filtered_remote = []
+        for param in remote:
+            filtered_param = {}
+            for k, v in param.items():
```

Comment:
```
if `ResolvedValue` in param:
    del param[`ResolvedValue`]
```

Or is mutating the dict here a problem?

Reply:
As a rule of thumb I don't like modifying the inputs.

Reply:
👍